src/hooks/useInterval.ts

```diff
@@ -1,27 +1,55 @@
-import { useEffect, useRef } from 'react'
+import { useEffect, useRef, useState, useCallback } from 'react'
+import isUndefinedOrNull from 'utils/isUndefinedOrNull'
+import useLastUpdated from './useLastUpdated'
 
-export default function useInterval(callback: () => void, delay: null | number, leading = true) {
-  const savedCallback = useRef<() => void>()
+export default function useInterval(
```

Comment:
Could you help me double check other components using `useInterval`?

We should keep the same behaviour for other places. To be safe, I think we could create another hook with better name.

Reply:
The only other place that useInterval used is in locked pools BurningCountdown. I tested it, it is working fine 👍